core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.ts

```diff
@@ -147,6 +147,7 @@ export class DotContentEditorComponent implements ControlValueAccessor, OnInit {
      */
     monacoInit(monacoEditor) {
         this.monacoEditors[monacoEditor.name] = monacoEditor.editor;
+        this.monacoEditors[monacoEditor.name].focus();
```

Comment:
Can we test this?

Reply:
let me check and update you about this.

Reply:
@fmontes can I add this test case in #23349 because I already write test cases of this component in this PR?

Reply:
Ok.